catalog/resource_lakehouse_monitor.go

```diff
@@ -12,11 +12,11 @@ import (
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
 )
 
-const lakehouseMonitorDefaultProvisionTimeout = 15 * time.Minute
+const qualityMonitorDefaultProvisionTimeout = 15 * time.Minute
 
 func WaitForMonitor(w *databricks.WorkspaceClient, ctx context.Context, monitorName string) error {
-	return retry.RetryContext(ctx, lakehouseMonitorDefaultProvisionTimeout, func() *retry.RetryError {
-		endpoint, err := w.LakehouseMonitors.GetByTableName(ctx, monitorName)
+	return retry.RetryContext(ctx, qualityMonitorDefaultProvisionTimeout, func() *retry.RetryError {
```

Comment:
NOTE: the Go SDK changes will go live with the next Go SDK release (currently scheduled for this Friday)